userspace/libscap/engine/gvisor/parsers.cpp

```diff
@@ -143,28 +171,23 @@ static parse_result parse_container_start(const char *proto, size_t proto_size,
 	}
 
 	// encode clone exit
-
-	ret.status = scap_event_encode_params(event_buf, &event_size, scap_err, PPME_SYSCALL_CLONE_20_X, 20,
-		(int64_t) 0, // child tid (0 in the child)
-		gvisor_evt.args(0).c_str(), // actual exe is not currently sent
-		scap_const_sized_buffer{args.data(), args.size()},
-		tid_field, // tid
-		tgid_field, // pid
-		(int64_t) 1, // ptid
-		"", // cwd
-		(uint64_t) 75000, // fdlimit
-		(uint64_t) 0, // pgft_maj
-		(uint64_t) 0, // pgft_min
-		0, // vm_size
-		0, // vm_rss
-		0, // vm_swap
-		gvisor_evt.args(0).c_str(), // comm
-		scap_const_sized_buffer{cgroups.c_str(), cgroups.length() + 1}, // cgroups
-		0, // clone_flags
-		context_data.credentials().real_uid(), // uid
-		context_data.credentials().real_gid(), // gid
-		(int64_t) 1, // vtid
-		(int64_t) 1); // vpid
+	ret.status = scap_gvisor::fillers::fill_event_clone_20_x(
+	                 event_buf, &event_size, scap_err,
+		         0,          // res = 0 in the child thread
+	                 exe.c_str(),
+	                 scap_const_sized_buffer{args.data(), args.size()},
+	                 tid_field,  // tid
+	                 tgid_field, // pid
+	                 1,          // ptid for initial process
+	                 "/",        // cwd for initial process
```

Comment:
Nit: this is actually different from the `cwd` value (`""`)  in the previous call to `scap_event_encode_params`, which also IIUC encodes process data for `init`.

Reply:
Good catch.
I thought I was doing the right thing, by hardcoding the current working directory as "/", for the container start event.  But now I'm not so sure.
I will restore the original value (empty string).